### PR TITLE
new data format

### PR DESCRIPTION
**Pull request by Matoo125**
https://epidemicforecasting.slack.com/archives/C0100EK7N8N/p1584532954213700

https://www.notion.so/Data-specs-b0a76e480f3c4eedacf7b9384ca3aa67

New JSON data format from @gavento

### CODE REVIEW

**Comment by Matoo125** [comment]
I wouldn't merge it until we have new data (I'm using only tmp dataset with 2 countries)

**Comment by Matoo125** [comment]
The only relevant code is this and bit of css

```
var svg = d3
  .select("#my_dataviz")
  .attr("class", "row")
  .style("width", "100%")
  .append("div")
  .attr("class", "col-md-9 col-sm-12")
  .append("svg")
  .attr("preserveAspectRatio", "xMinYMin meet")
  .attr("viewBox", "0 0 750 750")
  .classed("svg-content", true)
  .append("g")
  .attr("transform", "translate(" + margin.left + "," + margin.top + ")");

// Legend
d3.select("#my_dataviz")
  .append("div")
  .attr("class", "col-md-3 col-sm-12")
  .style("padding", "0px")
  .append("div")
  .attr("class", "legend")
  .append("div")
  .html(
    `<span class="color1">
      Weak seasonality
      <br/>
      Weak reduction in air travel
    </span>
    <br>
    <br>
    <span class="color2">
      Medium seasonality
      <br/>
      Weak reduction in air travel
    </span>
    <br>
    <br>
    <span class="color3">
      Strong seasonality
      <br/>
      Weak reduction in air travel
    </span>
    <br>
    <br>
    <span class="color41">
      Weak seasonality
      <br/>
      Strong reduction in air travel
    </span>
    <br>
    <br>
    <span class="color5">
      Medium seasonality
      <br/>
      Strong reduction in air travel
    </span>
    <br>
    <br>
    <span class="color6">
      Strong seasonality
      <br/>
      Strong reduction in air travel
    </span>`
  );
```

File: src/server/static/js/lines.js [hunk 1]
```diff
@@ -41,69 +41,72 @@ function getCountries(data) {
...
-      row["Cumulative Median_s=0.7_at=0.7"],
-      row["Cumulative Median_s=0.1_at=0.2"],
-      row["Cumulative Median_s=0.85_at=0.7"],
-      row["Cumulative Median_s=0.1_at=0.7"],
-      row["Cumulative Median_s=0.7_at=0.2"]
-    ]);
-  });
-  return combinations;
-}
+// function getCountryBetaData(data) {
```
Comment (wolverdude) [inline src/server/static/js/lines.js]: Please delete old code, don't just comment it out. This is what version control is for.

File: src/server/static/js/lines.js [hunk 2]
```diff
@@ -41,69 +41,72 @@ function getCountries(data) {
...
-  );
-  countryBetaData = selectedCountryBeta.items;
+function getListOfScenarios(activeData) {
+  return Object.keys(activeData)
+}
+
+
+
+
+d3.json("https://srv-file11.gofile.io/download/mKfSCQ/tmp.json")
```
Comment (wolverdude) [inline src/server/static/js/lines.js]: This line fails when I run it. Please update with a more permanent data source.

**Comment by wolverdude** [comment]
@Matoo125, should I go ahead and merge or is it not ready yet?

**Pull request merged**